cypress/integration/project_source_browsing.spec.js

```diff
@@ -9,14 +9,16 @@ before(() => {
 beforeEach(() => {
   cy.visit("./public/index.html#/profile/projects");
   cy.contains("platinum").click();
+  // Make sure there is enough time to load the commits which refreshes the horizontal menu
+  cy.wait(500);
```

Comment:
If possible we should avoid fixed `wait`s, because it makes the tests brittle.
A better approach is to check for the presence of an element on screen. This way cypress will wait only as long as necessary and is guaranteed to succeed.

In this case I'd add a `data-cy` to the commit counter and check that the content equals to the number of commits.

Something like: `cy.pick("commit-count").contains("14")`.